includes/Integrations/Core_Themes_Support.php

```diff
@@ -77,9 +105,29 @@ public function extend_theme_support() {
 	 */
 	public function embed_web_stories() {
 		$customizer = new Customizer();
+		$this->enqueue_style( $this->style_handle );
 		echo $customizer->render_stories(); // phpcs:ignore -- WordPress.Security.EscapeOutput.OutputNotEscaped - Escaped web stories HTML.
 	}
 
+	/**
+	 * Add a class if it is one of supported core themes.
+	 *
+	 * @since 1.3.0
+	 * @param array $classes Array of body classes.
+	 *
+	 * @return array Updated array of classes.
+	 */
+	public function add_core_theme_classes( $classes ) {
+
+		$classes[] = 'has-web-stories-support';
+
+		if ( $this->showing_customizer_stories ) {
+			$classes[] = 'is-showing-header-stories';
+		}
```

Comment:
As per my comment above, I think we can avoid having to add a `is-showing-header-stories` class